### PR TITLE
more osu! patterns

### PR DESCRIPTION
**Pull request by Tenexxt**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

(sorry for the commit spam, it wasn't intended i just started making this on the github website, resulting in many useless commits. if this is a problem, i can upload everything again into 1 commit)

If you find something that you thing should be changed, please tell me so i can look into it

### CODE REVIEW

File: wiki/Beatmap/Pattern/it.md
Comment (The-Last-Cookie) [inline wiki/Beatmap/Pattern/it.md:32]: Have you discussed this change with active Italian translators?

File: wiki/Beatmap/Pattern/osu!/Bursts/en.md
Comment (The-Last-Cookie) [inline wiki/Beatmap/Pattern/osu!/Bursts/en.md:9]:
```suggestion
A **burst** is a pattern where three or more [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on top of each other in 1/4 or faster ticks on the [timeline](/wiki/Client/Beatmap_editor/Timelines#hit-objects).
```

File: wiki/Beatmap/Pattern/osu!/Bursts/en.md
Comment (The-Last-Cookie) [inline wiki/Beatmap/Pattern/osu!/Bursts/en.md:10]: The transition between bursts and streams is fluid as far as my knowledge goes, but mentioning a general direction here might help improve the definition.

Something like:

There is no hard limit when bursts become streams but generally speaking, patterns with more than 10 consecutive hit objects are called streams.

File: wiki/Beatmap/Pattern/osu!/Death_Stream/en.md
Comment (The-Last-Cookie) [inline wiki/Beatmap/Pattern/osu!/Death_Stream/en.md:9]:
```suggestion
A **death stream** is a type of [stream](/wiki/Beatmap/Pattern/osu!/Stream) pattern. They occur in [Expert](/wiki/Ranking_criteria/Difficulty_naming) difficulties and test the players ability to click rapidly in a steady rhythm for longer periods.
```

File: wiki/Beatmap/Pattern/osu!/Death_Stream/en.md
Comment (The-Last-Cookie) [inline wiki/Beatmap/Pattern/osu!/Death_Stream/en.md:7]:
```suggestion
# Death stream
```

File: wiki/Beatmap/Pattern/osu!/Death_Stream/en.md
Comment (The-Last-Cookie) [inline wiki/Beatmap/Pattern/osu!/Death_Stream/en.md:11]:
```suggestion
**Death streams** tend to last longer than 30 [hit circles](/wiki/Gameplay/Hit_object/Hit_circle).
```

File: wiki/Beatmap/Pattern/osu!/csj/en.md
Comment (The-Last-Cookie) [inline wiki/Beatmap/Pattern/osu!/csj/en.md:7]: I think adding this as a subsection to `Jump` is okay.

File: wiki/Beatmap/Pattern/common/double/en.md
Comment (The-Last-Cookie) [inline wiki/Beatmap/Pattern/common/double/en.md:19]:
```suggestion
![](img/taikodouble.png "Visual representation of a double in the osu!taiko gamemode.")

```
Add a line break between two images.

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]
In my opinion, it's better if changes at this scale are done in separate PRs (one for English and one or more for translations *after* English has been merged). Otherwise, you will translate the article twice or the work was in vain if the maintainers decide against adding the article to the wiki. Furthermore, it saves time for translators of other languages.

Also, the folder names `common` and `double` do not adhere to the naming convention (first letter upper case).

File: wiki/Beatmap/Pattern/common/double/en.md
Comment (Tenexxt) [reply wiki/Beatmap/Pattern/common/double/en.md:19]: Will start working on fixes once I'm back home, thanks for helping

File: wiki/Beatmap/Pattern/common/en.md
Comment (Tenexxt) [reply wiki/Beatmap/Pattern/common/en.md:7]: While I agree that I can drop the common patters for now since there is only one for now, the description of the index pages were untouched and I only added the newer patterns to them, but I can update the description as well I suppose

File: wiki/Beatmap/Pattern/osu!/Death_Stream/en.md
Comment (Tenexxt) [reply wiki/Beatmap/Pattern/osu!/Death_Stream/en.md]: applied

File: wiki/Beatmap/Pattern/osu!/csj/en.md
Comment (Tenexxt) [reply wiki/Beatmap/Pattern/osu!/csj/en.md]: I thought of that, and i also think it fits there more.
Applied

File: wiki/Beatmap/Pattern/common/double/en.md
Comment (Tenexxt) [reply wiki/Beatmap/Pattern/common/double/en.md]: Since the common patters were deleted untill i get more of them, you could say this is applied?

File: wiki/Beatmap/Pattern/osu!/Death_Stream/en.md
Comment (Tenexxt) [reply wiki/Beatmap/Pattern/osu!/Death_Stream/en.md]: done

File: wiki/Beatmap/Pattern/common/en.md
Comment (Tenexxt) [reply wiki/Beatmap/Pattern/common/en.md]: dropped common patterns

File: wiki/Beatmap/Pattern/osu!/Jump/en.md
Comment (Akarinnnnn) [inline wiki/Beatmap/Pattern/osu!/Jump/en.md:14]: If this section is considered as a sub-section for `Jump`, more than one `#` is the suitable syntax(btw number of `#`s represent level, max 6).

File: wiki/Beatmap/Pattern/osu!/Stream/en.md
Comment (cl8n) [inline wiki/Beatmap/Pattern/osu!/Stream/en.md:24]: mention of expert diff is unnecessary / too specific

File: wiki/Beatmap/Pattern/osu!/Stream/en.md
Comment (cl8n) [inline wiki/Beatmap/Pattern/osu!/Stream/en.md:26]: avoid exact counts like this (too specific)

**Review by cl8n — CHANGES_REQUESTED** [review]
overall I don't think the approach taken here is very good, the idea should be to document the consensus (or lack thereof) on what these pattern names mean, rather than try to give them specific definitions. i think the existing sections for Jump and Stream are good examples of how to draw the line between universally-understood terminology and conventions that are common but not definitive

File: wiki/Beatmap/Pattern/osu!/Jump/en.md
Comment (Tenexxt) [reply wiki/Beatmap/Pattern/osu!/Jump/en.md:16]: The reason why I wanted for CSJ to be specific, or atleast to some extent, is because I feel that saying that it's a jump that goes across the screen is just not a good definition at all, and if so I'd like to give it even if it wouldn't be used in such way.
Since everyone plays on different hardware and settings, there can easily be a situation where a CSJ would not be actually go accros the screen.

**I could easily go from not specifying it in pixels, but I want for it to have more of a strict definition so people who play on different settings and new players can have a easy way to differ them.**

Also there are maps that mostly consist of higher spaced jumps that can be sometimes CSJ, so good luck differing one from another if you can't play the map and don't have any experience and any sort of definition to help you.

Unless it will be a really big requirement to drop it from the wiki, or just not giving it a any sort of a bit more expanded definition, I'm not planning to drop it entirely.

Comment (cl8n) [reply wiki/Beatmap/Pattern/osu!/Jump/en.md:16]: there is no one way to differentiate when to use certain adjectives to describe things and that's okay. I think you're getting kind of lost in this quest to define vague terms to the point of not actually giving the reader any valuable info about their real-world usage

**Pull request closed without merging**
